source/Octopus.Tentacle/Startup/OctopusProgram.cs

```diff
@@ -311,6 +312,14 @@ void InitializeLogging()
             LogManager.ThrowConfigExceptions = true;
             LogManager.Configuration = new XmlLoggingConfiguration(nLogFile);
 #endif
+            //if we are running in kubernetes, make the console log more helpful
+            if (PlatformDetection.Kubernetes.IsRunningInKubernetes)
+            {
+                var consoleLayout = new SimpleLayout("${longdate}  ${uppercase:${level}:padding=5}  ${message}${onexception:${newline}${exception:format=ToString}}");
```

Comment:
Do we need the date if Lens can add the timestamp for us anyway?

Reply:
Not necessarily, however the lens timestamps are slightly lower resolution (no milliseconds).

Also, not everyone uses Lens for viewing logs (in this team they do, but I know Cail uses K9s for his k8s stuff)

Reply:
Actually ... is this similar to @scme0 's change? https://github.com/OctopusDeploy/OctopusTentacle/pull/815/files/3f6991192d857afe7d0526aa5fe6ae09e986387a

Reply:
well he reverted it in his PR and I put it back it 😆 

I would argue there is some benefit in having it written in the log message and not requiring a 3rd party tool to add it

Reply:
Yep it is 😅 but happy with it either way. This code updates it only for Kubernetes Tentacle. Not sure if it's worth just updating the nlog config so it's the same for all Tentacles? Might be a bit cleaner.

Reply:
@scme0 I didn't want to change logging for non-k8s tentacles because I don't know how people use (or not use) those logs. At least this is segregated to just the k8s

Reply:
I would suggest attempting to use a variables like I did with log level as it's a bit clearer what's happening maybe?

Reply:
Taking with @flin-8, I'm going to have a different nlog file completely for k8s

Reply:
@scme0 @flin-8 I've changed this to use a custom nlog file for Kubernetes with the different log layout in 63aa6e0a